src/runtime/components/overlays/Notification.vue

```diff
@@ -8,12 +8,8 @@
             <UAvatar v-if="avatar" v-bind="{ size: ui.avatar.size, ...avatar }" :class="ui.avatar.base" />
 
             <div class="w-0 flex-1">
-              <p :class="ui.title">
-                {{ title }}
-              </p>
-              <p v-if="description" :class="ui.description">
-                {{ description }}
-              </p>
+              <p :class="ui.title" v-html="title" />
+              <p v-if="description" :class="ui.description" v-html="description" />
```

Comment:
```vue
<slot>
  <p :class="ui.title">
    {{ title }}
  </p>

  <p :class="ui.description">
    {{ description }}
  </p>
</slot>
```

Reply:
@Haythamasalama I see, thanks for the example, how would this work programmatically?

Reply:
```vue
<UNotification>
  <p>
   // your title 
  </p>
  <p>
   // your description  
  </p>
</UNotification>
```

Reply:
no I mean using the composable `useToast().add` [ here ](https://ui.nuxtlabs.com/overlays/notification#usage)